README.md

```diff
@@ -10,104 +10,157 @@
-* __Post actions__ - Create a GitHub issue from a post or attach a post message to an issue
-    * hover over a post to reveal the post actions menu and click the __More Actions (...)__ option
-* __Sidebar buttons__ - Stay up-to-date with how many reviews, unread messages, assignments and open pull requests you have with buttons in the Mattermost sidebar
+* __Daily reminders__ - The first time you log in to Mattermost each day, get a post letting you know what issues and pull requests need your attention.
+* __Notifications__ - Get a direct message in Mattermost when someone mentions you, requests your review, comments on or modifies one of your pull requests/issues, or assigns you on GitHub.
+* __Post actions__ - Create a GitHub issue from a post or attach a post message to an issue. Hover over a post to reveal the post actions menu and click **More Actions (...)**.
+* __Sidebar buttons__ - Stay up-to-date with how many reviews, unread messages, assignments, and open pull requests you have with buttons in the Mattermost sidebar
 * __Slash commands__ - Interact with the GitHub plugin using the `/github` slash command
-    * __Subscribe to a respository__ - Use `/github subscribe` to subscribe a Mattermost channel to receive notifications for new pull requests, issues, branch creation and more in a GitHub repository
-        - For instance, to post notifications for issues, issue comments, and pull requests matching the label `Help Wanted` from `mattermost/mattermost-server`, use
-
-          ```
-          /github subscribe mattermost/mattermost-server issues,pulls,issue_comments,label:"Help Wanted"
-          ```
-        - The following flags are supported:
-           - `--exclude-org-member`: events triggered by organization members will not be delivered. It will be locked to the organization
-            provided in the plugin configuration and it will only work for users whose membership is public. Note that organization members and collaborators
-            are not the same.
-    * __Get to do items__ - Use `/github todo` to get an ephemeral message with items to do in GitHub, including a list of unread messages and pull requests awaiting your review
-    * __Update settings__ - Use `/github settings` to update your settings for notifications and daily reminders
-    * __And more!__ - Run `/github help` to see what else the slash command can do
-
-## Installation
+
+## Before You Start
+
+This guide assumes that you have a GitHub account and that you're a Mattermost System Admin.
+
+If you’re running Mattermost v5.11 or earlier, first download the latest release of this plugin from the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-github/releases) and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
```

Comment:
@hanzei, is 5.11 unsupported? If so, we can either remove this section or we can have a version with 5.11 and a current version?